monitoring/bin/deploy_monitoring_viya.sh

```diff
@@ -48,8 +48,9 @@ set -e
 # Prometheus Pushgateway
 PUSHGATEWAY_ENABLED=${PUSHGATEWAY_ENABLED:-true}
 if [ "$PUSHGATEWAY_ENABLED" == "true" ]; then
-  PUSHGATEWAY_CHART_VERSION=${PUSHGATEWAY_CHART_VERSION:-1.11.0}
+  PUSHGATEWAY_CHART_VERSION=${PUSHGATEWAY_CHART_VERSION:-2.0.3}
   if helm3ReleaseExists prometheus-pushgateway $VIYA_NS; then
+    kubectl delete deploy -n $VIYA_NS -l app=prometheus-pushgateway
```

Comment:
Did you test this?  If you delete the deployment here, won't the next line (i.e. getting the svcClusterIP) fail?  And wouldn't that make the helm command, which uses that environment variable, also fail?

Also, even if the kubectl command accepts "deploy" as a valid value, let's spell out the full resource type (i.e. "deployment").

Reply:
Yes, this was tested.  Deleting the deployment did not delete the Service so the svcClusterIP variable still populated successfully.

I did push a change to spell out "deployment" so this should be good now.  Additionally, we do know the name of the deployment "prometheus-pushgateway" so I went ahead and updated to look for the specific name as opposed to the label that are put on the pods.